<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTestSetupForm</class>
 <widget class="QWidget" name="EditTestSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>994</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cantarell</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Solver Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxOnlyRequires">
            <property name="text">
             <string>Only Requires</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBoxForceResolve">
            <property name="text">
             <string>Force Resolve</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxAllowNameChange">
            <property name="text">
             <string>allowNameChange</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxAllowDowngrade">
            <property name="text">
             <string>allowDowngrade</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkBoxCleandepsOnRemove">
            <property name="text">
             <string>cleandepsOnRemove</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBoxallowVendorChange">
            <property name="text">
             <string>allowVendorChange</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBoxAllowArchChange">
            <property name="text">
             <string>allowArchChange</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxIgnorealreadyrecommended">
            <property name="text">
             <string>Ignore already recommended</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBoxDupAllowVendorChange">
            <property name="text">
             <string>dupAllowVendorChange</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxDupAllowArchChange">
            <property name="text">
             <string>dupAllowArchChange</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxDupAllowNameChange">
            <property name="text">
             <string>dupAllowNameChange</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBoxdupAllowDowngrade">
            <property name="text">
             <string>dupAllowDowngrade</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_Licencebit">
            <property name="text">
             <string>licencebit</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxshowMediaId">
            <property name="text">
             <string>show MediaId</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hardware Info</string>
         </property>
         <property name="buddy">
          <cstring>lineEditHardwareInfo</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Multiversion</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonAddMultiversion">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="3">
           <widget class="QListView" name="listViewMultiversion"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonRemoveMultiversion">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Sources</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonAddSource">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonRemoveSource">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QTableView" name="tableViewSources"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxSolverFocus"/>
       </item>
       <item row="6" column="2">
        <widget class="QGroupBox" name="groupBox_17">
         <property name="title">
          <string>Autoinst</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_30">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonAddAutoInst">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonRemoveAutoInst">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QListView" name="listViewAutoInst"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Modalias</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0" rowspan="3">
           <widget class="QListView" name="listViewModalias"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonAddModalias">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonRemoveModalias">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>SolverFocus</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxSolverFocus</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditHardwareInfo"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditSystemcheck"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>System Check</string>
         </property>
         <property name="buddy">
          <cstring>lineEditSystemcheck</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Locales</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonAddLocale">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonRemoveLocale">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QTableView" name="tableViewLocales"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Channels</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonAddChannel">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QTableView" name="tableViewChannels"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonRemoveChannel">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Force Install</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonAddForceInstall">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonRemoveForceInstall">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QTableView" name="tableViewForceInstall"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
